docs/UserGuide.md

```diff
@@ -437,8 +431,9 @@ Facing issues with SweetRewards? Here are some common problems and how to resolv
 
 ## Technical Issues
 1. Application won't start<br>
-🚫 **Problem**: Double-clicking the jar file doesn’t open SweetRewards.<br> 
-✅ **Solution**: Verify that Java 11 or higher is installed. Open a command prompt, navigate to the application's directory, and execute java -jar sweetrewards-v1.2.jar. If issues persist, re-download the jar file to rule out corruption.
+🚫 **Problem**: Double-clicking the jar file doesn’t open SweetRewards.<br>
+✅ **Solution**: Verify that Java 11 or higher is installed. Open a command prompt, navigate to the application's directory, and type in `java -jar sweetrewards-v1.2.jar` (Note the version number might differ, use the one that is seen in the version of SweetRewards that you have downloaded) and press enter. If issues persist, re-download the jar file to rule out corruption.
```

Comment:
I think can change `sweetrewards-v1.2.jar` to something like `sweetrewards-v<VERSION_NUM>` so that it is easier to update over iterations